cmd/zed/zed.d/history_event-zfs-list-cacher.sh.in

```diff
@@ -46,8 +46,14 @@ case "${ZEVENT_HISTORY_INTERNAL_NAME}" in
     set|inherit)
         # Only act if one of the tracked properties is altered.
         case "${ZEVENT_HISTORY_INTERNAL_STR%%=*}" in
-            canmount|mountpoint|atime|relatime|devices|exec| \
-                readonly|setuid|nbmand|encroot|keylocation) ;;
+            canmount|mountpoint|atime|relatime|devices|exec|readonly| \
+              setuid|nbmand|encroot|keylocation| \
+              org.open-zfs.systemd:noauto|org.open-zfs.systemd:requires| \
+              org.open-zfs.systemd:requires-mounts-for| \
+              org.open-zfs.systemd:before|org.open-zfs.systemd:after| \
+              org.open-zfs.systemd:wanted-by|org.open-zfs.systemd:nofail| \
```

Comment:
It sounds like @aerusso is looking to do wanted-by and required-by upstream. I suppose that makes sense, rather than just having one. So I suppose we should add an org.open-zfs.systemd:required-by too.

Reply:
Nit: I used `RequiredBy` and `WantedBy` in my PR (not `required-by`, in order to match the standard systemd names). I assume that will go through, but let's make sure the names settle before committing to anything.

Reply:
RequiresMountsFor becomes requires-mounts-for, so I think we're good with required-by and wanted-by: https://www.freedesktop.org/software/systemd/man/systemd.mount.html#x-systemd.requires-mounts-for=

Reply:
Yep, that looks about right, thanks! I'll modify my PR there.